test-requirements.txt

```diff
@@ -6,3 +6,4 @@ pytest-timeout~=1.2
 pytest-xdist~=1.15
 requests_mock~=0.7.0
 wheel~=0.29
+planemo~=0.36
```

Comment:
this should appear in `bear-requirements.txt` , and not in `test-requirements.txt`.

That will solve the error you can see at https://travis-ci.org/coala/coala-bears/jobs/195095556#L5596